server/game/cards/09.3-TSs/GeneNorthStar.js

```diff
@@ -0,0 +1,25 @@
+const DudeCard = require('../../dudecard.js');
+
+class GeneNorthStar extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Shootout: Gene North Star',
+            playType: ['shootout'],
+            cost: ability.costs.discardFromPlay(card => 
+                card.parent === this &&

```

Comment:
```suggestion
                this.equals(card.parent) &&
```

Reply:
Came from Mongwau The Mighty, patch applied.